### PR TITLE
imyogeshgaur-1-C++

### PR DESCRIPTION
**Pull request by imyogeshgaur**

### CODE REVIEW

File: Q1/imyogeshgaur/run.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+To run C++ program you have to install g++ compiler after installing it type the following command in your Terminal.
+
+<br/>
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/run.md:3]: Not required here. For more information regarding markdown format, check out [markdown-cheetsheet](https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md)

**Review by Yatharth0045 — CHANGES_REQUESTED** [review]

File: Q1/imyogeshgaur/Solution.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        if (first != last)
+            return false;
+        num = (num % div) / 10;
+        div = div / 100;
+    }
+    return true;
+}
+bool isPallinDrome(int sum)
+{
+    if (PallinDrome(sum))
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:21]: Please check the logic once again.
Hint: Not working with 1221

**Review by Yatharth0045 — CHANGES_REQUESTED** [review]

File: Q1/imyogeshgaur/Solution.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+using namespace std;
+bool PallinDrome(int num)
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:3]: Also, @imyogeshgaur , please correct the spelling

File: Q1/imyogeshgaur/Solution.cpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        int first = num / div;
+        int last = num % 10;
+        if (first != last)
+            return false;
+        num = (num % div) / 10;
+        div = div / 100;
+    }
+    return true;
+}
+bool isPallinDrome(int sum)
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:19]: Spell Check!! @imyogeshgaur

File: Q1/imyogeshgaur/Solution.cpp (hunk 2 above)
Comment (Yatharth0045) [reply Q1/imyogeshgaur/Solution.cpp:21]: Spell Check Here too @imyogeshgaur

File: Q1/imyogeshgaur/Solution.cpp [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+{
+    int N, sum = 0, div = 1;
+    cout << "Enter the Number : ";
+    cin >> sum;
+    while (N > 0)
+    {
+        sum += (N % 10);
+        N /= 10;
+    }
+    if (isPallinDrome(sum))
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:36]: SpellCheck @imyogeshgaur

File: Q1/imyogeshgaur/Solution.cpp [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        cout << "Pallindrome !!!";
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:37]: SpellCheck Again @imyogeshgaur

File: Q1/imyogeshgaur/Solution.cpp [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    else
+        cout << "Not a Pallindrome !!!";
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:39]: Here too @imyogeshgaur

**Review by Yatharth0045 — CHANGES_REQUESTED** [review]

File: Q1/imyogeshgaur/Solution.cpp [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    if (PallinDrome(sum))
+        return true;
+    else
+        return false;
+}
+int main()
+{
+    int N, sum = 0, div = 1;
+    cout << "Enter the Number : ";
+    cin >> sum;
```
Comment (Yatharth0045) [inline Q1/imyogeshgaur/Solution.cpp:30]: Are you sure about this that the input should be sum? @imyogeshgaur

**Review by Yatharth0045 — CHANGES_REQUESTED** [review]

**Comment by imyogeshgaur** [comment]
ok i will make the necessary changes

File: Q1/imyogeshgaur/Solution.cpp (hunk 8 above)
Comment (imyogeshgaur) [reply Q1/imyogeshgaur/Solution.cpp:30]: I think i have added it by mistake and not test it afterward but i will correct it 😊

Comment (imyogeshgaur) [reply Q1/imyogeshgaur/Solution.cpp:30]: thanks for the review 👍🏻 @Yatharth0045

**Pull request closed without merging**